src/python/dart/schema/base.py

```diff
@@ -25,7 +25,7 @@ def default_and_validate(model, schema):
         raise DartValidationException(str(best_match(errors)))
 
     user_id = "anonymous"
-    if current_user and current_user.email:
+    if current_user and hasattr(current_user, 'email'):
```

Comment:
In local mode when we test with authorization this code fails since there is no email attribute in the current_user object.  This change just checks that the email attribute exists.